README.md

```diff
@@ -11,47 +11,48 @@ Currently, the plugin is tracking the following data.
-|----------------------|-----------------------------------------------------|
-| jenkins.job.duration | Build duration (in seconds)                         |
-| jenkins.job.waiting  | Time spent waiting for job to run (in milliseconds) |
-
-
-List of metrics sent via Dogstatsd:
-Requires the Datadog Agent to be installed on the Jenkins host.
 
 | Metric Name              | Description                                                   |
 |--------------------------|---------------------------------------------------------------|
-| jenkins.job.completed    | Jobs Completed                                                |
+| jenkins.job.completed    | Jobs Completed rate                                           |
+| jenkins.job.started      | Jobs Started rate                                             |
 | jenkins.job.leadtime     | Lead Time                                                     |
 | jenkins.job.cycletime    | Cycle Time                                                    |
 | jenkins.job.mttr         | MTTR: time between last failed job and current successful job |
 | jenkins.job.feedbacktime | Feedback time from code commit to job failure                 |
 | jenkins.job.mtbf         | MTBF, time between last successful job and current failed job |
+| jenkins.job.duration     | Build duration (in seconds)                                   |
+| jenkins.job.waiting      | Time spent waiting for job to run (in milliseconds)           |
+| jenkins.queue.size       | Queue Size                                                    |
+| jenkins.scm.checkout     | SCM checkout rate                                             |
+
 
 List of service checks:
 * Build status (jenkins.job.status)
 
 All events, metrics, and service checks include the following tags, if they are available:
 * `job`
 * `result`
-* (git) `branch` (Available when using the [Git Plugin](https://wiki.jenkins.io/display/JENKINS/Git+Plugin))
+* (Git Branch, SVN revision or CVS branch) `branch` 
+  * Git Branch available when using the [Git Plugin](https://wiki.jenkins.io/display/JENKINS/Git+Plugin)
+* `node`
 
-Optional tags, included in events, metrics, and service checks. (Toggle from `Manage Jenkins -> Configure System`)
-* `node` (disabled by default)
 
 ## Customization
 From the global configuration page, at `Manage Jenkins -> Configure System`.
 * Blacklisted Jobs
-	* A comma-separated list of job names that should not monitored. (eg: susans-job,johns-job,prod_folder/prod_release).
+	* A regex comma-separated list of job names that should not be monitored. (eg: susans-job,johns-.*,prod_folder/prod_release).
+* Whitelisted Jobs
+	* A regex comma-separated list of job names that should be monitored. (eg: susans-job,johns-.*,prod_folder/prod_release).
+* Global Tags
+	* A list of jobs and comma-separated list of tags that should be applied for that job (eg: (.*?)_job_(*?)_release, owner:$1, release_env:$2, optional:Tag3).
```

Comment:
Hmm not clear at all from the sentence (the example says it all in the contrary) 
maybe something like `a regex to match a job, and a list of tags to apply to that job, all separated by a comma. tags can reference match groups in the regex using the $ symbol`